config/config.go

```diff
@@ -82,6 +82,7 @@ const (
 	EnvThreeScaleServicePort                = "THREESCALE_SERVICE_PORT"
 	EnvTracingEnabled                       = "TRACING_ENABLED"
 	EnvTracingInClusterURL                  = "TRACING_IN_CLUSTER_URL"
+	EnvTracingNamespaceSelector             = "TRACING_NAMESPACE_SELECTOR"
```

Comment:
This can be deleted - we no longer have env vars for all config settings (we only use env vars for sensitive values like passwords).